### DISCUSSION TITLE
GitHub Team invitation

### DISCUSSION THREAD

**Discussion by jzempel** [renovatebot/renovate#12577] [Request Help]
Is it possible to invite [renovate-bot](https://github.com/renovate-bot) to an existing GH team? In [our](https://github.com/zendeskgarden) case, certain teams have permission to perform demo deployments via CI. Currently we need to follow-up on Renovate PRs with a manual deploy that the bot is [unauthorized](https://app.circleci.com/pipelines/github/zendeskgarden/ckeditor?branch=renovate%2Fckeditor-ckeditor5-basic-styles-31.x) to perform.

**Comment by rarkins** [comment]
Yes, it's a regular account so you can probably invite it, then someone from our side with access would need to find the invitation and accept it.

**Reply by jzempel** [reply]
sweet 🙌 . invite sent.

**Reply by andrewpollock** [reply]
Did this ever get actioned? We're in a similar boat today, where we've invited the account to have read-level access to avoid needing to manually trigger Google Cloud Build runs on every PR/PR refresh. I'm wondering if the invite will be accepted without further action on our end, or of I need to surface it here...

**Reply by rarkins** [reply]
Are you sure you still need it? Previously Renovate signed all its commits using the @renovate-bot *user* but nowadays all commits are made by the Renovate *app*. So unless you're using the Forking Renovate app, you shouldn't see @renovate-bot anywhere

**Reply by andrewpollock** [reply]
> Are you sure you still need it? Previously Renovate signed all its commits using the @renovate-bot *user* but nowadays all commits are made by the Renovate *app*. So unless you're using the Forking Renovate app, you shouldn't see @renovate-bot anywhere

The problem we're trying to solve is needing to explicitly allow our Google Cloud Run action to run with a "/gcbrun" comment because the PR isn't originating from a contributor

**Reply by rarkins** [reply]
Are you using github.com/apps/renovate (i.e. the main Renovate app)?

In this case the PR creator and committer aren't "real" users on GitHub. They'll show up as something like `29139614+renovate[bot]@users.noreply.github.com`, which is a non-existing email address

**Reply by andrewpollock** [reply]
> Are you using github.com/apps/renovate (i.e. the main Renovate app)?
> 
> In this case the PR creator and committer aren't "real" users on GitHub. They'll show up as something like `29139614+renovate[bot]@users.noreply.github.com`, which is a non-existing email address

I believe we're using the main Renovate app. Here's a sample PR: https://github.com/google/osv.dev/pull/859

**Reply by rarkins** [reply]
Actually, that's Forking Renovate, so that's why you're seeing the real user (you need a real user and not an app user in order to fork repos). In this case if you invite the account @renovate-bot to any repos or orgs then please notify me here or via email and I can check that account to accept the invite. We normally don't log into or monitor that account regularly because it's used for "production"

**Reply by andrewpollock** [reply]
I have invited the account already.

**Reply by jzempel** [reply]
@rarkins I have also re-invited @renovate-bot. Can you accept when you get the chance? TY

**Reply by rarkins** [reply]
I have accepted both

**Reply by andrewpollock** [reply]
@rarkins we've also invited it to https://github.com/google/osv-scanner, could you accept that one too please?

**Reply by andrewpollock** [reply]
@rarkins I've just invited it to https://GitHub.com/Google/osv-scanner-action as well, could you accept that one too please

**Reply by rarkins** [reply]
Done
